api/api.go

```diff
@@ -37,6 +39,43 @@ type SubmissionInfo struct {
 	State string `json:"state"`
 }
 
+// StatusInfo contains information about a user's status on a particular language track.
+type StatusInfo struct {
+	Language string `json:"track_id"`
+	Recent   struct {
+		Problem   string `json:"problem"`
+		Submitted string `json:"submitted_at"`
+	}
+	Fetched []string `json:"fetched"`
+	Skipped []string `json:"skipped"`
+}
+
+func (s *StatusInfo) String() {
+	if len(s.Fetched) == 0 && s.Recent.Problem == "" {
+		fmt.Println("\nYou have yet to begin the", s.Language, "track!")
+	} else {
```

Comment:
In Go the idiom is to ditch the else if there is a return in the `if` block (or if you could have a return in the if block). This minimizes indentation. [Effective Go](https://golang.org/doc/effective_go.html#if) says:

> In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.

Here there's no behavior that happens after you've printed this line, which would let you minimize the indentation in the next part.